backend/LexBoxApi/Controllers/ProjectController.cs

```diff
@@ -222,6 +222,18 @@ public async Task<ActionResult<int>> UpdateLexEntryCount(string code)
         return result is null ? NotFound() : result;
     }
 
+    [HttpPost("updateAllLexEntryCounts")]
+    [AdminRequired]
+    public async Task<ActionResult> UpdateAllLexEntryCounts(bool onlyUnknown)
```

Comment:
```suggestion
    public async Task<ActionResult> UpdateAllLexEntryCounts(bool onlyUnknown = true)
```

Reply:
Done in commit d56bc4210cac4fe8a31a748b1eeb931416d2fd5c along with some other changes (see below).